src/panel/spoiler_card/spoilerCardActions.js

```diff
@@ -14,29 +12,17 @@ function isValidTitle(index, title) {
 	if (commonActions.isInvalidTitle(title) || commonActions.isDuplicateTitleSkipIndex(title, index)) {
 		return false;
 	}
-	else {
-		return true;
-	}
+	
+	return true;

```

Comment:
Sorry, you mean the revised return statement is readable, or the current version?